### ISSUE TITLE
Non Root theming on android 12 requires uses `android.theme.customization.system_accent` along with `android.theme.customization.system_palette`, used by newer versions

### ISSUE THREAD

**Issue by MinekPo1** [Mahmud0808/ColorBlendr#48]
ROM: Motorola Android 12

to be honest not entirely sure what is going on, should probably look deeper into it

**Comment by MinekPo1** [comment]
yup, it does work. I did that before even creating the issue, I was mostly unsure of what the difference between the two was.

I did some experiments and digging , here is what I've found:

- [the source code for reading this setting](https://android.googlesource.com/platform/frameworks/base/+/refs/tags/android-12.1.0_r1/packages/SystemUI/src/com/android/systemui/theme/ThemeOverlayController.java#527)
- `android.theme.customization.system_palette` controls the tint of neutral colors
- `android.theme.customization.accent_color` controls the hue of accent colors

looking at the main branch however, only `android.theme.customization.system_palette` is used (along with `android.theme.customization.theme_style`), so this only affects older versions of android (AOSP 13 also does not have this).

Also it seems like setting `android.theme.customization.color_source` is unnecessary on both the AOSP 12 branch and the main branch, as both will treat the option being missing as equal to `"preset"` (or more specifically different from `"home_wallpaper"` and `"home_wallpaper"`). Note also that on the AOSP 12 branch, monet code is in `./packages/SystemUI/monet/src` rather than in `./packages/SystemUI/src/com/android/theme`.
